pyproject.toml

```diff
@@ -64,6 +64,22 @@ randomize = true
 [[tool.hatch.envs.hatch-test.matrix]]
 python = ["3.10", "3.11", "3.12"]
 
+[tool.hatch.envs.bench]
+description = """Custom environment for running benchmarks.
+Use `hatch run +py=3... bench:generate ${args}` to generate test models.
+Use `hatch run +py=3... bench:serialize ${args}` to benchmark serialization code.
```

Comment:
out of curiosity whats does the `+py=3...` cover that the python matrix below doesn't?

Reply:
For `hatch run` if you don't select from the matrix it would run in all combinations from the matrix, one after the other. The `+py=3.11`, etc. options allow selecting the python version, so you'd run only once.